docs/LV_client/tutorials/Logger.md

```diff
@@ -0,0 +1,45 @@
+# Mono_logger.vi
```

Comment:
Everything here is well written and clear. Just a few styling things, the list of sensors are not showing as a bulleted lists on the build, try adding an empty line above the start of the bulleted list should correct the styling. Also, probably don't need to add a number before the starting paragraph.

Reply:
Looks better! 
![Screen Shot 2020-09-04 at 5 00 42 PM](https://user-images.githubusercontent.com/41026694/92287710-2c2d5c80-eed0-11ea-9fbe-ea5e6afd5a75.png)
showing up twice in side panel though